embassy-stm32/src/usart/mod.rs

```diff
@@ -1010,6 +1015,11 @@ impl<'d, M: Mode> UartRx<'d, M> {
         }
         Ok(())
     }
+
+    /// Set baudrate
+    pub fn set_baudrate(&self, baudrate: u32) -> Result<(), ConfigError> {
```

Comment:
These impls should also be extended to `Buffered*` and `RingBuffered*` structs

Reply:
Thanks for the feedback, just did that. Please take a look